identity-service/src/notifications/notificationQueue.js

```diff
@@ -1,6 +1,17 @@
 const { deviceType } = require('./constants')
 const { drainMessageObject: sendAwsSns } = require('../awsSNS')
 const { sendBrowserNotification, sendSafariNotification } = require('../webPush')
+const racePromiseWithTimeout = require('../utils/racePromiseWithTimeout.js')
+const { logger } = require('../logging')
+
+// guessed a value
+const SEND_NOTIF_TIMEOUT_MS = 5000 // 5 sec
```

Comment:
lmk any thoughts on this value

Reply:
IMO, something like 20s

Reply:
kk

Reply:
For reference, I just timed a single call to post to aws sns - was about 400 ms (From my machine)

Reply:
yeah 20s seems high to me, but will defer to you two

Reply:
it generally makes sense to keep number very high - goal is just to not infinitely stall, since we currently don't have retries if for some reason calls become very slow we'll end up skipping a ton of these